providers/module.rb

```diff
@@ -29,7 +32,7 @@ def whyrun_supported?
 action :remove do
   execute "semodule-remove-#{new_resource.name}" do
     command "/usr/sbin/semodule -r #{new_resource.name}"
-    only_if "/usr/sbin/semodule -l | grep '^#{new_resource.name}\\s'"
+    only_if "/usr/sbin/semodule -l | grep -w '#{new_resource.name}'"
```

Comment:
`-w` is new to me. However:
```bash
echo 'bla blu' | grep -w 'blu'
```
Passes, so don't like it, as I don't want to introduce any (further) holes in my detection logic.